### PR TITLE
[YUNIKORN-137] Update examples and docs to use docker images from ASF repo

### PR DESCRIPTION
**Pull request by kingamarton**

### CODE REVIEW

File: docs/developer-guide.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ The docker image built from previous step has embedded some important build info
 these info with docker `inspect` command.
 
 ```
-docker inspect yunikorn/yunikorn-scheduler-k8s:latest
+docker inspect yunikorn/yunikorn:scheduler-latest
```
Comment (yangwwei) [inline docs/developer-guide.md:71]: shouldn't this be `apache/yunikorn:scheduler-latest`?

Comment (kingamarton) [reply docs/developer-guide.md]: you are right! Just changed it

**Comment by wilfred-s** [comment]
On line 59 we point to the yunikorn repo should that be changed as part of this update:
```
Currently, there are some published docker images under [this docker hub repo](https://hub.docker.com/u/yunikorn), you are free to fetch and use.
```

**Comment by yangwwei** [comment]
Hi @kingamarton 

I did a scan and there is more to update:

```
➜  incubator-yunikorn-k8shim git:(master) ✗ grep -nr "yunikorn-scheduler-k8s" ./*
./README.md:74:This command will build a binary executable with version `latest` and the docker image tag is `yunikorn/yunikorn-scheduler-k8s:latest`.
./README.md:79:docker inspect --format='{{.Config.Labels}}' yunikorn/yunikorn-scheduler-k8s:latest
./deployments/scheduler/scheduler.yaml:35:        - name: yunikorn-scheduler-k8s
./deployments/scheduler/scheduler.yaml:36:          image: yunikorn/yunikorn-scheduler-k8s:latest
./deployments/scheduler/scheduler-load.yaml:37:        - name: yunikorn-scheduler-k8s
./deployments/scheduler/scheduler-load.yaml:38:          image: yunikorn/yunikorn-scheduler-k8s:latest
./helm-charts/yunikorn/templates/deployment.yaml:43:        - name: yunikorn-scheduler-k8s
./helm-charts/yunikorn/values.yaml:26:  repository: yunikorn/yunikorn-scheduler-k8s
./landmark/docker_images.yaml:19:  yunikorn-scheduler-k8s: __registry__/yunikorn-scheduler-k8s
```
We need to do the same for admission, and web docker images. Could you please check? Thanks

**Comment by kingamarton** [comment]
> Hi @kingamarton
> 
> I did a scan and there is more to update:
> 
> ```
> ➜  incubator-yunikorn-k8shim git:(master) ✗ grep -nr "yunikorn-scheduler-k8s" ./*
> ./README.md:74:This command will build a binary executable with version `latest` and the docker image tag is `yunikorn/yunikorn-scheduler-k8s:latest`.
> ./README.md:79:docker inspect --format='{{.Config.Labels}}' yunikorn/yunikorn-scheduler-k8s:latest
> ./deployments/scheduler/scheduler.yaml:35:        - name: yunikorn-scheduler-k8s
> ./deployments/scheduler/scheduler.yaml:36:          image: yunikorn/yunikorn-scheduler-k8s:latest
> ./deployments/scheduler/scheduler-load.yaml:37:        - name: yunikorn-scheduler-k8s
> ./deployments/scheduler/scheduler-load.yaml:38:          image: yunikorn/yunikorn-scheduler-k8s:latest
> ./helm-charts/yunikorn/templates/deployment.yaml:43:        - name: yunikorn-scheduler-k8s
> ./helm-charts/yunikorn/values.yaml:26:  repository: yunikorn/yunikorn-scheduler-k8s
> ./landmark/docker_images.yaml:19:  yunikorn-scheduler-k8s: __registry__/yunikorn-scheduler-k8s
> ```
> 
> We need to do the same for admission, and web docker images. Could you please check? Thanks

@yangwwei those files are already handled in https://github.com/apache/incubator-yunikorn-k8shim/pull/116, since they are part of the k8shim repo.

**Comment by yangwwei** [comment]
Oops I messed them up. Thanks for taking care of it. LGTM.

**Pull request merged**